gcp/api/server.py

```diff
@@ -781,21 +815,23 @@ def _datastore_normalized_purl(purl: PackageURL):
 
 
 @ndb.tasklet
-def query_by_commit(
-    context: QueryContext,
-    commit: bytes,
-    to_response: Callable = bug_to_response) -> tuple[list, ndb.Cursor]:
+def query_by_commit(context: QueryContext,
+                    commit: bytes,
+                    to_response: Callable = bug_to_response) -> list:
   """Query by commit."""
```

Comment:
Please improve this docstring while you're here: https://google.github.io/styleguide/pyguide.html#s3.8.3-functions-and-methods

Reply:
Added a bunch of docs to all those functions, though I feel like I probably missed some exceptions that could be raised. Also fixed the typing a bit on the arguments.